de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -418,6 +546,26 @@ protected void undo(SWTBotTreeItem item) {
 		waitForEditingDomainAndUiThread();
 	}
 	
+	/**
+	 * Undo the last command made
+	 */
+	protected void undoCommand() {

```

Comment:
To stay consistent with our other API (we already have a method "undo" targeting SWTBotTree items) we should just call this method undo as well.